Tests/instance_notifier.py

```diff
@@ -124,7 +113,7 @@ def slack_notifier(slack_token, secret_conf_path, server, user, password, build_
 
 if __name__ == "__main__":
     options = options_handler()
-    if options.nightly:
+    if options.instance_tests:
         with open('./Tests/instance_ips.txt', 'r') as instance_file:
```

Comment:
switch to using`env_results.json`

Reply:
also remove the condition on line 99 and then trigger the "instance tests" build again